src/layouts/learn-more.svelte

```diff
@@ -0,0 +1,125 @@
+<script context="module">
+  export const linkTitleMap = {
```

Comment:
It would be clearer to use a plain array of objects to represent the pages, e.g.:
```ts
const links = [
    { slug: "terms", title: "Terms of Services" },
];
```